Task_functions/index.js

```diff
@@ -1,11 +1,15 @@
 //Write code for the 1st task below this
```

Comment:
The function name should be 'addTwo' instead of 'addTWo' to follow proper naming conventions.